packages/baml/baml_client/sync_client.ts

```
@@ -35,263 +42,322 @@ import { DO_NOT_USE_DIRECTLY_UNLESS_YOU_KNOW_WHAT_YOURE_DOING_CTX, DO_NOT_USE_DI
+			const raw = this.runtime.callFunctionSync(
+				"AnthropicCompletion",
+				{
+					prompt: prompt,
+				},
+				this.ctxManager.cloneContext(),
+				options.tb?.__tb(),
+				options.clientRegistry,
+				collector,
+			);
+			return raw.parsed(false) as string;
+		} catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)